.github/workflows/release.yml

```diff
@@ -9,19 +9,20 @@ jobs:
   release:
     name: Release docs
     runs-on: ubuntu-18.04
+    timeout-minutes: 30
     steps:
     - name: Check out code
       uses: actions/checkout@v2
       with:
-        fetch-depth: 0
+        fetch-depth: 1
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
         go-version: '~1.15'
     - name: Build docs
       run: make build.public
       env:
-        HUGO_BASE_URL: https://thethingsstack.io/
+        HUGO_BASE_URL: /docs
```

Comment:
This needs to be the full URL, so `https://www.thethingsindustries.com/docs`

Reply:
Why? Site seems to be working with just `/docs`. Otherwise how do we serve docs to both `thethingsindustries.com` and `preview.thethingsindustries.com`?

Reply:
We need it to generate valid canonical URLs for search engines

Reply:
Can you explain?

Reply:
We render a `<link rel="canonical" ...>` on every page, so that search engines know what the "main" page is, and what pages are aliases (and should therefore not be used in search results). These canonical URLs need to be fully qualified, so including `https://www.thethingsindustries.com/docs`.